cmd/lnmuxd/run.go

```diff
@@ -157,7 +157,7 @@ func run(ctx context.Context, cfg *Config, group *errgroup.Group) error {
 			grpc_zap.UnaryServerInterceptor(log.Desugar()),
 		)
 		streamInterceptors = append(streamInterceptors,
-			grpc_zap.StreamServerInterceptor(log.Desugar()),
+			grpc_zap.StreamServerInterceptor(log.Desugar()), //nolint: contextcheck
```

Comment:
Add //nolint: contextcheck otherwise the linter will detect a problem we can't solve... 
https://github.com/bottlepay/lnmux/actions/runs/3090768021/jobs/4999993946